game-2.py

```diff
@@ -1,4 +1,10 @@
 from crpg import generate
+from game_loader import start
 
-game = generate("game-2-layout.dl")
-game.start()
+dl = start()
+if( str(dl) != "-1"):
+    game = generate(dl)
+    game.start()
+else:
+    print("Exiting")
+    exit()
```

Comment:
Is this file still needed with the current setup?

Reply:
No, the game can be launch by calling the game_loader.py directly. I just wanted to leave this in working condition in case anyone elses stuff needed it.

Reply:
In that case, I would revert it to the previous commit, that way the example shows how the API can be used